x-pack/plugins/alerting/server/rules_client_factory.ts

```diff
@@ -185,6 +185,9 @@ export class RulesClientFactory {
         }
         return { apiKeysEnabled: false };
       },
+      isSystemAction(actionId: string) {
```

Comment:
I named it `isSystemAction` and not `isSystemActionConnector` because the checks in the alerting plugin will be done upon rule actions.